extensions/summary/summary.md

```diff
@@ -0,0 +1,85 @@
+
+ * A "search" extension should have a way to describe which properties can be used in sorting.
+
+ * An "editing" extension should have a way to describe which properties are writeable (e.g. in an update) or might want to allow for a different schema for each method (e.g. one schema for create, one for update).
+
+## Example
+
+Here is an example of what this summary resource might look like for a "buildings" collection (e.g. at https://example.com/collections/buildings/summary)
+
+```json
+{
+  "properties": {
+    "application/geo+json": {
+      "height": {
+        "schema": {
+          "href": "http://example.com/collections/buildings/schema#/definitions/height"
+        }
+      },
+      "age": {
+        "schema": {
+          "href": "http://example.com/collections/foo/schema#/definitions/age"
+        },
+        "filter": {
+          "disallowed": true
+        },
+        "search": {
+          "unsortable": true
+        }
+      },
+      "taxID": {
+        "schema": {
+          "href": "http://example.com/collections/foo/schema#/definitions/age"
+        },
+        "editing": {
+          "readonly": true
+        }
+      }
+    }
+  }
+}
+```
+
+Notes on the above example:
+
+ * The properties of the `properties` object (ack!) are content types.  Using an object means that there will be one member per content type.  So it looks like the responses `content` object in OpenAPI 3.
+
+ * The `schema` property would be defined by the core Features spec.  In the case of JSON schema, an href could deeply link to the spceific definition.  In the case of an XSD doc, there could be two properties: `href` and `path` (maybe?) so an XPath expression could be provided to refer to the part of the schema specific to a single property.
+
+ * Individual extensions could add their own members to each of the properties object.  For example, the "filter" expression could define a `filter` object.
+ 
+ * I like the idea that defaults are `false` - so the absence of a property implies the default property.  This means that by default, a property would be sortable for example.  If a service wanted to disallow sorting on a particular property, they would add `"unsortable": true` to change the default behavior.
+
+## Alternatives
+
+In discussion with others here at the Features/STAC sprint, we discussed a few other alternatives.  This isn't a comprehensive set of notes, but we covered things like the list below:
+
+### A new resource per extension
+
+We considered the alternative that each extension would add its own resource at a new path.  For example, the filter extension might add a `/collections/buildings/queryables` resource.  Then the search extension might add a `/collections/buildings/sortables` resource.  And then the editing extension might add a `/collections/buildings/writeables` resource.  This began to feel unscalable and awkward for clients who want to get a complete picture of how to work with a collection.
+
```

Comment:
and those various capabilities would still have a need to point to the schema, so a lot of redundancy